tests/test_client.py

```diff
@@ -18,13 +19,13 @@ def test_not_called(self):
     def test_get_server_time(self, monkeypatch):
         self.client.get_server_time()
         self.mock_requests.assert_called_once_with(
-            CurrencyComConstants.SERVER_TIME_ENDPOINT
```

Comment:
Tests do not pass. You should pass constants as an object not a class.
It would be better to replace `CurrencyComConstants` with `self.client.constants`